### PR TITLE
{lang}[GCCcore/10.2.0] Rust v1.52.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/Rust/Rust-1.52.1-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://static.rust-lang.org/dist/']
+sources = ['rustc-%(version)s-src.tar.gz']
+checksums = ['3a6f23a26d0e8f87abbfbf32c5cd7daa0c0b71d0986abefc56b9a5fbfbd0bf98']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('CMake', '3.18.4'),
```
Comment (boegel) [inline easybuild/easyconfigs/r/Rust/Rust-1.52.1-GCCcore-10.2.0.eb:18]: `Python` and `pkg-config` are missing as a build dependency here

We don't have a `bare` Python, just using the regular one is fine here imho (we can't do that in more recent toolchains, where `Rust` has become a build dependency for `Python`...)

**Review by boegel — CHANGES_REQUESTED** [review]
@sassy-crick see https://github.com/sassy-crick/easybuild-easyconfigs/pull/5

File: easybuild/easyconfigs/r/Rust/Rust-1.52.1-GCCcore-10.2.0.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/r/Rust/Rust-1.52.1-GCCcore-10.2.0.eb:18]: Thanks for spotting that! 
I have seen the PR you opened, once the tests have passed I will merge it with this one here.

Comment (sassy-crick) [reply easybuild/easyconfigs/r/Rust/Rust-1.52.1-GCCcore-10.2.0.eb:18]: Merged.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2606.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/de06d1518ea9e35c21b1c84994dd20f8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3617.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/3b632287098f5ce5dec8df8b76b1d193 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**